ci/check-style.sh

```diff
@@ -31,8 +31,11 @@ cd "${PROJECT_ROOT}"
 # below to time blocks of the script. A newline is automatically included.
 readonly TIMEFORMAT="... %R seconds"
 
+# To use the printf command rather than the shell builtin.
```

Comment:
You may want to add _"which makes a difference when ..."_.

Reply:
Done.